ouroboros-consensus-test/test-consensus/Test/Consensus/BlockchainTime/Simple.hs

```diff
@@ -373,6 +374,100 @@ instance MonadDelay (OverrideDelay (IOSim s)) where
 instance MonadDelay (OverrideDelay IO) where
   threadDelay d = OverrideDelay $ ReaderT $ \_schedule -> threadDelay d
 
+newtype OverrideDelaySTM m a = OverrideDelaySTM {
+      unOverrideDelaySTM :: ReaderT (StrictTVar m Schedule) (STM m) a
+    }
+
+deriving instance Alternative (STM m) => Alternative (OverrideDelaySTM m)
+deriving instance Applicative (STM m) => Applicative (OverrideDelaySTM m)
+deriving instance Functor     (STM m) => Functor     (OverrideDelaySTM m)
+deriving instance Monad       (STM m) => Monad       (OverrideDelaySTM m)
+deriving instance MonadPlus   (STM m) => MonadPlus   (OverrideDelaySTM m)
+
+instance (MonadSTM m, MonadThrow (STM m)) => MonadThrow (OverrideDelaySTM m) where
+    throwIO = OverrideDelaySTM . lift . throwIO
+    bracket before after f =
+         OverrideDelaySTM $ ReaderT $ \schedule ->
+          bracket       (runReaderT (unOverrideDelaySTM  before)   schedule)
+                  (\a -> runReaderT (unOverrideDelaySTM (after a)) schedule)
+                  (\a -> runReaderT (unOverrideDelaySTM (f a))     schedule)
+
```

Comment:
Style: There's an extraneous newline here.

Reply:
ok I'll change that, we often use 2 new lines as a break in `ouroboros-network`.